src/FsYacc.Core/fsyaccast.fs

```diff
@@ -1212,21 +1221,21 @@ let CompilerLalrParserSpec logf (spec: ProcessedParserSpec) : CompiledSpec =
 
 (*
 
-let example1 = 
-  let e = "E" 
+let example1 =
+  let e = "E"
   let t = "Terminal"
   let plus = "+"
   let mul = "*"
   let f = "F"
   let lparen = "("
   let rparen = ")"
   let id = "id"
-  
+
   let terminals = [plus; mul; lparen; rparen; id]
   let nonTerminals = [e; t; f]
-  
+
   let p2 = e, (NonAssoc, ExplicitPrec 1), [NonTerminal e; Terminal plus; NonTerminal t], None
-  let p3 = e, (NonAssoc, ExplicitPrec 2), [NonTerminal t], None in  
+  let p3 = e, (NonAssoc, ExplicitPrec 2), [NonTerminal t], None in
```

Comment:
Those are either by fantoms or my editor, I can revert if needed.

Reply:
As this is in a code comment, this is your editor's doing.